setSecretKey.ps1

```diff
@@ -0,0 +1,60 @@
+# Randomly generate a secret key. ONLY USE THIS FOR DEVELOPMENT!
+
+# Copied from http://dbadailystuff.com/2013/03/11/get-hash-a-powershell-hash-function
+function Get-Hash
```

Comment:
No need for any of this @WeilonYing, just use a secret key like `development-only` for development - security isn't important on a local stack when it's not exposed to real users.

I'd suggest removing this file entirely as it just complicates the local development process.

See what I did for Bark:

https://github.com/csesoc/bark-server/blob/master/config.py

I use the Flask config class pattern, and have different options for each - local dev uses hardcoded fake keys (and makes required ones optional i.e. `UDB_PASSWORD`). You'd be able to adopt a similar pattern in this django app.

Reply:
Ah right. I've adopted your pattern and added it to pull request #4. Thanks so much for the advice Nick :)